src/PlatformEngines.jl

```diff
@@ -130,11 +132,6 @@ function probe_symlink_creation(dest::AbstractString)
     end
 end
 
-# Global variable that tells us whether tempdir() can have symlinks
-# created within it.
-tempdir_symlink_creation = false
```

Comment:
Every time a `global` is removed from a file, an angel sheds a single tear of joy.